app/models/background.rb

```diff
@@ -67,4 +67,16 @@ def store_summary
   def compute_stdev(sum, sum2, n)
     Math.sqrt((sum2 - sum**2 / n.to_f) / (n - 1))
   end
+
+  def self.viewable(user)
+    if user.admin?
+      all
+    else
+      where("project_id in (select project_id from projects_users where user_id=?)
```

Comment:
make it where(project_id: Project.viewable(user).pluck(:id)) in case project.viewable changed in the future?